### PR TITLE
Golwallakoala

### PR DESCRIPTION
**Pull request by mgolwalla**

### CODE REVIEW

File: public/assets/sponsors/sponsor_us_back.svg
Comment (minsooerickim) [inline public/assets/sponsors/sponsor_us_back.svg:1]: the backside shouldn't have the shadoww 😓. u have to go in and manually delete the shadow, but it seems like nothing changed on the backside? Frontside is the only thing that needs to be fixed right?

**Review by minsooerickim — CHANGES_REQUESTED** [review]

File: public/assets/sponsors/sponsor_us_back.svg
Comment (minsooerickim) [reply public/assets/sponsors/sponsor_us_back.svg:1]: image

thats what it looks like if we leave the shadow in (it wont jump 😭)

Comment (mgolwalla) [reply public/assets/sponsors/sponsor_us_back.svg:1]: shadow gone! changed the back cause it had this typo 
image

Comment (mgolwalla) [reply public/assets/sponsors/sponsor_us_back.svg:1]: changed mentor front as well, was missing "kyoto, japan"
image

Comment (minsooerickim) [reply public/assets/sponsors/sponsor_us_back.svg:1]: awesome awesome 🥳

**Pull request merged**